contracts/interchain-token-service/tests/execute.rs

```diff
@@ -314,6 +314,159 @@ fn execute_message_interchain_transfer_should_scale_the_amount_when_source_decim
+        },
+        source_its_contract.clone(),
+        hub_message,
+    ));
+
+    let amount: nonempty::Uint256 = Uint256::from_u128(1000000000000000000u128)
+        .try_into()
+        .unwrap();
+
+    // send from chain 0 to chain 1
+    let transfer = InterchainTransfer {
+        token_id,
+        source_address: HexBinary::from([1; 32]).try_into().unwrap(),
+        destination_address: HexBinary::from([2; 32]).try_into().unwrap(),
+        amount,
+        data: None,
+    };
+    let hub_message = HubMessage::SendToHub {
+        destination_chain: configs[1].0.clone(),
+        message: transfer.into(),
+    };
+
+    assert_ok!(utils::execute_hub_message(
+        deps.as_mut(),
+        CrossChainId {
+            source_chain: configs[0].0.clone(),
+            message_id: router_message.cc_id.message_id.clone()
+        },
+        source_its_contract.clone(),
+        hub_message,
+    ));
+
+    let scaling_factor = Uint256::from_u128(10)
+        .checked_pow(configs[0].3.abs_diff(configs[1].3).into())
+        .unwrap();
+    let scaled_amount = amount.clone().checked_div(scaling_factor).unwrap();
+
+    // send back from destination to source
+    let transfer = InterchainTransfer {
+        token_id,
+        source_address: HexBinary::from([2; 32]).try_into().unwrap(),
+        destination_address: HexBinary::from([1; 32]).try_into().unwrap(),
+        amount: scaled_amount.try_into().unwrap(),
+        data: None,
+    };
+
+    let hub_message = HubMessage::SendToHub {
+        destination_chain: configs[2].0.clone(),
+        message: transfer.into(),
+    };
+
+    assert_ok!(utils::execute_hub_message(
+        deps.as_mut(),
+        CrossChainId {
+            source_chain: configs[1].0.clone(),
+            message_id: router_message.cc_id.message_id.clone()
+        },
+        source_its_contract.clone(),
+        hub_message,
+    ));
+
+    let scaling_factor = Uint256::from_u128(10)
+        .checked_pow(configs[0].3.abs_diff(configs[2].3).into())
+        .unwrap();
+    let scaled_amount = amount.clone().checked_div(scaling_factor).unwrap();
+
+    // send back from destination to source
+    let transfer = InterchainTransfer {
+        token_id,
+        source_address: HexBinary::from([2; 32]).try_into().unwrap(),
+        destination_address: HexBinary::from([1; 32]).try_into().unwrap(),
+        amount: scaled_amount.try_into().unwrap(),
+        data: None,
+    };
+
+    let hub_message = HubMessage::SendToHub {
+        destination_chain: configs[0].0.clone(),
+        message: transfer.into(),
+    };
+
+    let response_to_destination = assert_ok!(utils::execute_hub_message(
+        deps.as_mut(),
+        CrossChainId {
+            source_chain: configs[2].0.clone(),
+            message_id: router_message.cc_id.message_id.clone()
+        },
+        source_its_contract,
+        hub_message,
+    ));
+
+    goldie::assert_json!(json!({"response_to_destination": response_to_destination}));
```

Comment:
```suggestion
    goldie::assert_json!(response_to_destination);
```
Don't think you need `json!` when you have only one object to assert.